bundle/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Flamingos2Epics.scala

```diff
@@ -99,26 +100,34 @@ object Flamingos2Epics {
 
 }
 
-object Flamingos2EpicsInitializer {
+object Flamingos2Epics extends EpicsSystem {
+  
   val Log = Logger.getLogger(getClass.getName)
   val CA_CONFIG_FILE = "/Flamingos2.xml"
+  
+  // Still using a var and null, but at least now it's hidden. Attempts to access the single instance will
+  // now result in an Exception with a meaningful message, instead of a NullPointerExecption
+  private var instanceInternal: Flamingos2Epics = null
+  lazy val instance: Flamingos2Epics = {
+    if (instanceInternal == null) {
+      throw new Exception("Attempt to reference Flamingos2Epics single instance before initialization.")
+    }
+    instanceInternal
+  }
```

Comment:
Yes, I should have used an Option. I'll fix it.
The reason I wanted to treat Flamingos2Epics as a singleton is that it is not allowed to have more than one instance, because that means more than one object will try to control the instrument. But I could create and destroy the single instance on demand.